release-notes.md

```diff
@@ -1254,11 +1254,11 @@ follow what dcrwallet is doing, there is a log window below.  In the future,
 we may add better messaging that will allow updates to the mixing process 
 instead of just showing raw logs.
 
-Once privacy is enabled we have restricted external spending to the mixed 
-account.  This is to ensure privacy is not broken by spending from any unmixed 
-account.  Additionally, we have restricted address generation to the mixed 
-account, since only funds that have been properly mixed should be allowed to 
-end up there.
+Once privacy is enabled spending to external addresses is only allowed from the 
```

Comment:
Consider adding a comma for clarity: "Once privacy is enabled, spending to..."

Reply:
Added.